### DISCUSSION TITLE
Как к имеющему списку прибавить еще значение?

### DISCUSSION THREAD

**Discussion by Re-Requeste** [totumonline/totum-ru-issues-and-discussions#541] [Как это сделать?] [answered]
Здравствуйте, есть селект (множественный), формате ["1", "2", "4"] и т.п. Как, в этот селект добавть допустим "5" - id , не потеряв имеющие данные в селекте ?

![123](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/68dffac6-f55e-4e73-bc57-c0a653255169)

Пробовал так: 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/c1f2c209-4d6a-4a36-94c4-937ca450659a)

Выдает ошибка, что лист2 должен быть row/list

**Comment by erpic-ru** [comment, marked as answer]
поправьте 

` =: listAdd(list: $selectted; item: #h_user)`

upd: ну и селект нужно на селектлист заменить

**Reply by Re-Requeste** [reply]
Спасибо, упустил
